src/lib/control_allocation/control_allocation/ControlAllocationPseudoInverse.hpp

```diff
@@ -46,6 +46,7 @@
 #pragma once
 
 #include "ControlAllocation.hpp"
+#include <px4_platform_common/log.h>
```

Comment:
I guess this isn't necessary, as you don't enable it due to 
```c++

``` // adding #include <px4_platform_common/log.h> + PX4_WARN leads to failed linking on test

Reply:
Removed the requested line. Let's get it merged? 😃